src/scripts/slurm_status.py

```diff
@@ -344,7 +355,12 @@ def get_finished_job_stats(jobid):
                     factor = 1024 * 1024 * 1024
                 elif value[-1] == 'P':
                     factor = 1024 * 1024 * 1024 * 1024
-                    return_dict["ImageSize"] += int(value.strip('KMGTP')) * factor
+                elif value[-1] == 'K':
+                    factor = 1
```

Comment:
Note `factor = 1` is already the default

Reply:
Indeed, I just wanted an `elif` for the 'K' value so it couldn't fall into the else.